src/containers/Settings.jsx

```diff
@@ -114,12 +113,40 @@ class Settings extends React.Component {
 
   render() {
     const { organization } = this.props.data
+    const {optOutMessage } = organization
+    const formSchema = yup.object({
+      optOutMessage: yup.string().required()
+    })
+
     return (
       <div>
         <Card>
           <CardHeader
             title='Settings'
           />
+          <CardText>
+            <div className={css(styles.section)}>
+
+            <GSForm
+              schema={formSchema}
+              onSubmit={this.props.mutations.updateOptOutMessage}
+              defaultValue={{ optOutMessage }}
+            >
+
+              <Form.Field 
+                label='Default Opt-Out Message' 
+                name='optOutMessage'  
```

Comment:
can we add the prop `fullWidth` here so the user can see the full message (versus having to scroll within the form field) ?